lib/cputemp/service.py

```diff
@@ -93,10 +95,16 @@ def register_app_error_callback(self, error):
 
     def register(self):
         adapter = BleTools.find_adapter(self.bus)
+        if not adapter:
+            logger.error("No Bluetooth Adapter")
```

Comment:
Update these logs to say `Unable to start Bluetooth application: No Bluetooth adapter`